.github/workflows/mac.yml

```diff
@@ -17,7 +17,7 @@ concurrency:
 
 env:
   PYTHON_VERSION: '3.10'
-  OV_BRANCH: master
+  OV_BRANCH: 345163f87953fb0dd8dd590257eb7fc84378da8e
```

Comment:
aligned with commit ID used in OpenVINO provider (see https://github.com/openvinotoolkit/openvino.genai/commit/5ab58ca70dd2774595ad82768074c7a497aa9377#diff-6687d618bafed728bcf9516ca380234fa98f2b2d08f802085371646df236e495R47) because of issue https://github.com/openvinotoolkit/openvino/pull/28312